lib/zclient.c

```diff
@@ -554,6 +554,28 @@ void zclient_send_dereg_requests(struct zclient *zclient, vrf_id_t vrf_id)
 	}
 }
 
+int zclient_send_router_id_add(struct zclient *zclient, afi_t afi,
```

Comment:
let's create a generic function that both the _add and _delete function call.  cut-n-paste code is not good